### ISSUE TITLE
Mislabeled Variable - Yellowbrick's Occupancy Detection dataset

### ISSUE THREAD

**Issue by navarretedaniel** [DistrictDataLabs/yellowbrick#662]
**Describe the bug**
The 'CO2' (Carbon Dioxide) variable in Yellowbrick's version of the [Occupancy](https://github.com/DistrictDataLabs/yellowbrick/blob/47826d93ca4b912e9ef590c82d2a86e535d6a129/yellowbrick/datasets/download.py#L79) dataset is mislabeled as 'C02' (C-Zero-Two).

Like the Bike Sharing dataset referenced in the [Quick Start Walkthrough](http://www.scikit-yb.org/en/latest/quickstart.html#walkthrough "Yellowbrick Quick Start Walkthrough") and as noted in the [Example Datasets](http://www.scikit-yb.org/en/latest/api/datasets.html?highlight=occupancy), I believe that the source for this Occupancy Detection dataset is based upon the one uploaded to the [UCI Machine Learning Repository](https://archive.ics.uci.edu/ml/datasets/Occupancy+Detection+).

Looking at [utils.py](https://github.com/DistrictDataLabs/yellowbrick/blob/47826d93ca4b912e9ef590c82d2a86e535d6a129/yellowbrick/datasets/utils.py#L53), it appears as though the dataset is [locally hosted](https://s3.amazonaws.com/ddl-data-lake/yellowbrick/occupancy.zip) by Yellowbrick, so I suspect that the problem may lie in how this source file labels this variable in the meta.json and occupancy.csv files.

**Documentation Affected**
Updates to the [Scatter Plot Visualizer](http://www.scikit-yb.org/en/latest/api/contrib/scatter.html?highlight=C02) , [RadViz Visualizer](http://www.scikit-yb.org/en/latest/api/features/radviz.html?highlight=C02), [Classification Report](http://www.scikit-yb.org/en/latest/api/classifier/classification_report.html?highlight=C02), [Cross Validation Scores](http://www.scikit-yb.org/en/latest/api/model_selection/cross_validation.html?highlight=C02), [ROCAUC](http://www.scikit-yb.org/en/latest/api/classifier/rocauc.html?highlight=C02),  [Feature Importances](http://www.scikit-yb.org/en/latest/api/features/importances.html?highlight=C02), [Manifold Visualization](http://www.scikit-yb.org/en/latest/api/features/manifold.html?highlight=C02), [Contributing](http://www.scikit-yb.org/en/latest/contributing.html?highlight=C02), [Parallel Coordinates](http://www.scikit-yb.org/en/latest/api/features/pcoords.html?highlight=C02), and [FAQ](http://www.scikit-yb.org/en/latest/faq.html?highlight=C02) pages would be necessary.

Most of these are minor in that code snippets specifically reference "C02" rather than "CO2", but in a few occasions, the accompanying visual plots would also need to be updated.

For example, the [Scatter Plot Visualizer](http://www.scikit-yb.org/en/latest/api/contrib/scatter.html?highlight=C02) reference "C02" in its code snippets as well as its visual example:

* the features of interest:

``` python
# Load the classification data set
data = load_data("occupancy")

# Specify the features of interest and the classes of the target
features = ["temperature", "relative humidity", "light", "C02", "humidity"]
classes = ["unoccupied", "occupied"]

# Extract the numpy arrays from the data frame
X = data[features]
y = data.occupancy
```

* defining 'y':
``` python
from yellowbrick.contrib.scatter import ScatterVisualizer

visualizer = ScatterVisualizer(x="light", y="C02", classes=classes)

visualizer.fit(X, y)
visualizer.transform(X)
visualizer.poof()
```

* the corresponding Scatter Plot title and y-axis label:
![](http://www.scikit-yb.org/en/latest/_images/scatter.png)

**To Reproduce**

Yellowbrick:
```python
from yellowbrick.datasets import load_occupancy
import pandas as pd

data = pd.DataFrame(load_occupancy())

data.columns.tolist()
```

**Dataset**
* [Yellowbrick](https://s3.amazonaws.com/ddl-data-lake/yellowbrick/occupancy.zip)
* [UCI Machine Learning Repository](https://archive.ics.uci.edu/ml/datasets/Occupancy+Detection+)

**Expected behavior**
This dataset should reference "CO2" rather than "C02".

**Desktop:**
 - OS:  Windows 10
 - Python Version: 3.6
 - Yellowbrick Version:  0.9

**Comment by lwgray** [comment]
@navarretedaniel Great catch!  Thanks for taking the time to write such a detailed issue.  We will definitely look into this. @Kautumn06 can you comment on this?

Happy Holidays!

**Comment by Kautumn06** [comment]
Thank you @navarretedaniel for writing such a thoughtful and detailed issue! We are currently overhauling Yellowbrick's datasets so I am going to add this issue to Pull Request #535 so that this typo will be fixed in the updated `yellowbrick.datasets` module and throughout the documentation.

We are always looking for new [contributors](http://www.scikit-yb.org/en/latest/contributing.html) so please don't hesitate to reach out to us again if there is [an issue](https://github.com/DistrictDataLabs/yellowbrick/issues) or a new visualizer that you'd like to work on!

**Comment by bbengfort** [comment]
Thanks @navarretedaniel for the bug report; it's great to see the current cohort already participating in YB development!

**Comment by bbengfort** [comment]
@navarretedaniel just wanted to let you know, we made the fix and it will be updated in the v1.0 release!
